playground/backend/pkg/executors/executor.go

```diff
@@ -13,19 +13,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package executors
 // Interface for all executors (Java/Python/Go/SCIO)
```

Comment:
`// Interface for all executors (Java/Python/Go/SCIO)` could be removed since we have line 20.

Reply:
Done, it happened because of merge conflict resolving